hera_cal/hera_cm.py

```diff
@@ -0,0 +1,198 @@
+import numpy as np
+import pandas as pd
+import os
+import aipy
+import pyuvdata.utils as uv_utils
+from astropy.time import Time
+from dateutil.parser import parse
+
+cm_p_m = 100
```

Comment:
For consistency should probably be in aipy.const.  For now a comment has been added.